source/backend/api/Areas/CompensationRequisition/Controllers/CompensationRequisitionController.cs

```diff
@@ -124,9 +124,12 @@ public IActionResult GetCompensationRequisitionPayee([FromRoute] long id)
                 DateTime.Now);
             _logger.LogInformation("Dispatching to service: {Service}", _compensationRequisitionService.GetType());
 
-            var compensationPayee = _compensationRequisitionService.GetPayeeByCompensationId(id);
+            // TODO:
+            //var compensationPayee = _compensationRequisitionService.GetPayeeByCompensationId(id);
+            var compensationPayee = string.Empty;
 
-            return new JsonResult(_mapper.Map<CompensationPayeeModel>(compensationPayee));
+            //return new JsonResult(_mapper.Map<CompensationPayeeModel>(compensationPayee));
+            return new JsonResult(string.Empty);
```

Comment:
I don't know where this is used in the FE but I'm assuming it will get fixed in another PR??

Reply:
Yes, as noted on the pr description, the payee functionality requires additional fixes